src/app/handlers/command.handler.ts

```diff
@@ -37,6 +38,22 @@ export class CommandHandler extends Handler {
       return;
     }
 
+    // Check if a slash command of the same name exists, if so redirect to it
+    const slashCommand = commands.get(command.name);
+
+    if (slashCommand) {
+      const cachedCommand = this.container.guildService
+        .get()
+        .commands.cache.find((c) => c.name === command.name);
+
+      if (cachedCommand) {
```

Comment:
Whats the use case where we would need to double check that the slash command is in the fs **and** thats its deployed?

Shouldnt every thing in that map be deployed?